vendor/gopkg.in/check.v1/benchmark.go

```diff
@@ -1,9 +1,9 @@
 // Copyright (c) 2012 The Go Authors. All rights reserved.
-// 
+//
```

Comment:
Please revert this. Afaik that doesn't trigger any linter error and it could potentially cause merging conflicts when an update is made. Not worth it.